centreon/tests/e2e/cypress/e2e/Dashboards/07-dashboard-manage-permissions.feature

```diff
@@ -1,23 +1,28 @@
+@REQ_MON-22191
 Feature: Managing dashboard permissions
```

Comment:
Could that be: https://centreon.atlassian.net/browse/MON-18519 ? 
@tuntoja @paul-oureib

Reply:
yes it is